apps/emqx_durable_storage/src/proto/emqx_ds_proto_v3.erl

```diff
@@ -0,0 +1,141 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+-module(emqx_ds_proto_v3).
```

Comment:
I haven't initiate this series of modules myself, but I think the reasoning here is that this is the proto module for the `emqx_ds` _application_.  See [`emqx_bridge_proto_v1`](https://github.com/emqx/emqx/blob/a10f33870395fe5232ffc2d56e7b326ac29a5352/apps/emqx_bridge/src/proto/emqx_bridge_proto_v1.erl#L98) for example: it calls 2 modules other than `emqx_bridge`.

I guess for `emqx` application we made a finer proto module breakdown due to the increased complexity of that application.

Reply:
I think it was my mistake not to call this module `emqx_ds_replication_layer_proto_...` from the very beginning, because technically this BPAPI is specific to the builtin replication layer indeed. Not sure if we can change this easily now, though.